app/src/main/java/com/example/xpvehicles/interfaces/ParentAdapter.java

```diff
@@ -0,0 +1,39 @@
+package com.example.xpvehicles.interfaces;
+
+import android.view.View;
+import android.widget.Adapter;
+import android.widget.TextView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.xpvehicles.models.RentVehicle;
+import com.example.xpvehicles.models.Vehicle;
+
+import java.util.List;
+
+public interface ParentAdapter {
```

Comment:
Why do we always use interface? instead of regular class?

Reply:
Java classes can only have  1 superclass, but can extend from multiple interfaces, so it allows us to get around this restriction